tests/test_mssqlcliclient.py

```diff
@@ -202,6 +203,37 @@ def test_stored_proc_multiple_result_sets(self):
         finally:
             shutdown(client)
 
+    
+    def test_select_result_unicode_encoded(self):
+        """
+            Verify the column names and string values in rows returned by select statement are properly encoded as unicode.
+        """
+        db_name = 'db_' + random_str()
+        table_name = 'table_' + random_str()
+        try:
+            client = create_mssql_cli_client()
+            setup_query = u"CREATE DATABASE {0};"\
+                u"USE {0};"\
+                u"CREATE TABLE {1} (컬럼1 nvarchar(MAX), 컬럼2 int);"\
+                u"INSERT INTO {1} VALUES (N'테스트1', 1);"\
+                u"INSERT INTO {1} VALUES (N'테스트2', 2);"\
+                .format(db_name, table_name)
```

Comment:
Can we run these on a temporary table to assure that table is cleaned up when the connection closes?

Reply:
I added drop table statement for the clean up.

Reply:
If I look on the Adventure works database, I see a bunch of leaked resources from the mssql-cli tests. A temporary table would prevent this.

---
In reply to: [322518262](https://github.com/dbcli/mssql-cli/pull/278#discussion_r322518262) [](ancestors = 322518262)

Reply:
I have no idea how to convert this sql script into create-temporary-table script.
```
CREATE TABLE table_name (컬럼1 nvarchar(MAX), 컬럼2 int);
INSERT INTO table_name VALUES (N'테스트1', 1);
INSERT INTO table_name VALUES (N'테스트2', 2);
```
Please let me know.

Reply:
Just add a `#` prefix: https://docs.microsoft.com/en-us/sql/t-sql/statements/create-table-transact-sql?view=sql-server-2017#temporary-tables

Reply:
fixed.